src/gatsby-theme-carbon/components/Footer.js

```diff
@@ -4,33 +4,29 @@ import Footer from 'gatsby-theme-carbon/src/components/Footer';
 const Content = () => (
   <>
     <p>
-      By importing the <strong>Footer</strong> component from
-      gatsby-theme-carbon, we can supply our own props.
+      Have questions? Open an issue in <a href="https://github.com/carbon-design-system/carbon-for-ibm-website/issues/new/choose">Github</a>.
```

Comment:
@JennySanchez was thinking we should remove the email address, not sure if we're going with that.

cc: @wonilsuhibm @ljcarot